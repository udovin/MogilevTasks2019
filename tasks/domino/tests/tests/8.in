421
-61288191 17695
-47496889 57719
-50450941 18602
-79578117 118327
-65283836 80251
-49491507 114676
-55665599 3320
-60991039 29210
-48207226 33753
-68433732 53735
-76577828 7842
-47227552 41640
-78942302 62654784
-63018872 64784
-76685925 14480
-55824777 16400
-54167423 26968634
-80196046 46501
-50998837 6160
-52904335 42052
-70174284 7198
-71490514 26377
-74770514 7256
-52721650 106382
-48940312 350
-51944043 10089010
-54608134 53060
-62272551 20891
-53623798 40464599
-61648064 43260
-48404212 17290
-56086394 9844
-61589321 31073
-67072478 144777
-62366138 95421
-57195575 56454
-51841370 5979
-66629440 50735
-74323851 3659
-68276624 24882
-76704393 17087
-64612629 103826
-48114114 62874
-70905813 51161
-55787574 13787
-47040382 3848
-76011062 90903
-69738197 71789700
-59210437 9694
-66177862 22219
-79369476 14496
-73467448 50761
-54955052 64563
-69361868 27117
-67217384 66027
-77794175 100582
-59278195 45841
-64153058 107668
-48270723 1846
-74579695 28745
-64049478 9605
-50644461 11052
-61471813 20878
-59194750 15195
-77389160 24377
-47835731 4899
-60789519 9712
-81680187 12008
-76982536 49197
-77280827 38550
-74426375 16227
-71244348 47748
-57781821 14189
-57272238 57586556
-67288867 19032
-58714339 105529
-51813956 16139
-82044616 49670
-69418911 44762
-59099599 7351
-66279148 38323
-49770522 2626
-70756361 52038
-57395911 110579
-49775380 15031
-60159472 60373
-47142771 81798
-67538782 54507665
-77684328 68330
-81062140 34690
-47564863 31149
-75330813 60652
-49234532 17581724
-54724581 50863
-74276179 5278
-55032298 43186
-79877709 24092
-47776277 15090
-73608516 8271
-59349075 93228
-73627795 55496
-77325765 4890
-72439959 23514
-60590908 2094
-69887691 65463
-57835462 34675
-57632273 57743
-64866513 141364
-76813169 59969
-68146214 6676
-77198834 31997
-70628347 26900
-75219289 90509
-57026824 4196
-56580613 87133
-53869090 26461
-61167851 10630
-77964751 80224
-68264060 8982
-71431003 37326
-78514721 26228
-68339520 15448
-49183109 35651
-58536275 45197
-75711837 10273
-69304218 30648
-78778271 123613
-75948059 6826
-76195685 102056
-77894983 1316
-53964152 42518
-52855143 43272
-68918378 35504
-63164633 71636
-54502387 51964
-75430354 6472
-82014734 228
-63512296 70532
-65011851 113797
-47029374 6751
-53586900 29370
-49453831 12875
-50300206 103948
-70537102 86627
-52571613 24374
-49397245 54064
-55855185 40118
-73875487 17968
-72683979 114930
-82019174 23170
-51287114 15310
-66716030 60956
-58411971 83799
-64301126 82317
-71649615 2610
-60239531 40831
-60727732 39816
-74332738 39515
-61367394 37260
-46689163 76587
-79926545 8944
-58821434 75130
-48287991 60888
-57698389 54701
-50748718 57801
-54301948 7576
-58264782 82571389
-81693732 137708
-64729886 122086
-78187519 94552
-74616429 35739
-75830639 83778
-47738801 15421
-59627791 69080
-68684533 34913792
-62697170 45891
-79068228 64059
-80425563 11491
-75444934 94837
-57937807 7308
-78708520 55333
-80313149 6535
-74859347 48216
-80324655 15961
-63300534 79009062
-57517470 1142
-68819397 5651910
-78429627 65401
-48589017 13874
-80466817 73352
-75559015 50612
-69152045 52186
-48703012 82519
-77609835 21932
-60471246 6548
-51636103 21548
-65525430 2801
-71915870 69808
-53033875 3743
-52093981 8404
-72883875 11300
-47350363 101301
-81445631 70601
-67006214 53324
-66074368 40628
-47967118 91811
-67822033 3727
-50874550 75380
-70198295 81004622
-80603182 131037
-52326988 36826
-60867374 57122
-51760922 30885
-72152274 107087
-71654454 123101
-82188100 1638
-67434997 59591
-55993967 10781
-60711664 8041
-54364558 67689
-70719075 26553
-71799105 46786
-49890972 76549
-62519075 7499
-53080766 33246
-49151964 16174
-72925562 76468
-56901694 54078
-58820740 332
-64444473 92552
-57531858 80604
-80931632 62640
-78304127 36603
-50539781 45644
-77671449 7686
-55164947 146793
-48535090 3235
-49647411 20611
-47879935 61819
-61570726 12702
-52189205 109263
-62129944 75467
-57762061 4941
-77526832 61464
-57095743 2177034
-55449461 14177
-56133016 24951
-65933711 5014
-49798339 42542
-78062873 911
-81313602 19650
-71559155 40215
-58007906 91660
-70339568 63376
-81155514 155553
-81013039 40149
-73172595 42375910
-73729942 8100
-60010665 1784
-65669912 20421
-80739004 54502
-49062286 8265
-58635802 76497
-76623465 1534
-65187568 40598
-54828872 77822
-73573368 25277
-70506860 1137
-49373023 7902
-51135136 148804
-80170712 14937
-68209709 6065
-71039851 88059
-76877826 22122
-81565910 2089
-73331195 102
-50811705 9298
-74061239 87910
-74954883 79141
-74915887 37224
-81549553 15380
-81603361 52308
-50142607 105099
-52558079 5182
-73049516 55784
-69513588 14482
-76315633 74449
-68567356 7369
-54028473 71384981
-60371605 97833
-73296102 4902
-50532387 2914
-56754039 12790
-78764148 9899
-60837592 16453
-78918770 754
-67831265 74965
-62798226 77070
-66876086 56202
-61878552 723
-76786662 7869
-52629120 54120
-76900572 4776
-53205088 1557
-53821041 6335
-55322601 20939
-79215169 21048
-75739982 81586
-60604483 73084
-80071169 4730
-65136392 23068
-46944010 17257
-59886972 6412
-55709795 44960
-55757322 5624
-47631976 17574
-55522350 45277
-81840916 20351
-48377492 1827
-66386802 77470
-71164099 78996
-52482614 6753
-67695381 38898
-61762298 101665
-66474225 148136
-61926808 60685
-46786849 33515
-65507072 12751
-55768232 5452
-49071255 30455
-50063317 19932
-50642219 1236
-63818000 47720
-56304199 95312
-77048462 17351
-72805484 27741
-80864320 5173
-76575819 1777
-53302841 8913
-63441120 51308
-58151789 14646
-60303193 56553
-48430162 46279
-79464372 99065
-61051130 72464050
-63774474 7436
-76567350 8118
-53137290 23553
-51540101 57046
-69470829 17343
-74155364 10083
-52490191 20454
-72288521 20169
-58203123 41416
-51896853 7416
-71351605 32127
-60806946 29100
-57999642 4051
-49739030 22468
-62891903 34427497
-61423259 27412
-70043945 58259
-68172099 16035
-54126844 17551
-53734876 65892
-80088608 50477
-73919079 15396
-68971796 53126
-58946096 79958
-58260611 1913
-69659254 74398
-67283517 5038
-70458720 21082
-49481592 4788
-56891528 7126
-63621145 75430
-53472812 82227
-68513448 14494
-56668778 14542
-68026862 28095
-72009643 139370
-59718149 19270
-66871181 2361
-51915032 9795
-65396501 45617
-50025438 28262
-48920925 1728
-78604987 24749
-64561121 7798
-69060930 88837
-81884552 9584
-75072287 52078
-61987996 56076
-60117719 40119
-56163533 63049
-53316277 55520
-55685284 235
-56453495 122754
-72531254 33692
-51402381 83098
-65530539 116660
-76425221 108090
-63895953 75703
-59783217 42412
-76143752 13662
-51891117 3958
-73957966 79132
-57961371 19212
-65780221 101196
-79731259 59349
-48807417 90199
-56777350 18567
-61166197 596
-73333299 52668
-60021543 31165
-62620642 42258
-70137254 21734
-67934713 8207451
-59481523 67474
-62357460 5915
-53015467 8434
