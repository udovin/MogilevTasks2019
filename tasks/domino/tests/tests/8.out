1 1 1 3 1 5 1 1 1 1 2 1 385 1 1 1 98 2 1 1 1 1 1 2 1 70 1 1 91 1 1 1 2 2 3 1 1 1 1 2 1 2 1 1 1 1 2 278 1 1 1 1 1 1 1 1 1 2 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 134 2 2 1 3 1 1 1 1 2 1 2 2 1 251 2 1 1 1 34 1 1 1 1 1 1 2 3 1 1 1 1 1 1 4 2 1 1 1 1 1 1 1 2 2 1 1 1 1 2 1 1 1 3 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 2 2 2 1 1 2 1 1 1 1 1 1 2 2 1 1 1 2 2 1 1 1 150 4 3 1 1 1 1 1 266 1 1 1 2 1 1 1 1 2 210 1 267 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 1 2 283 1 1 2 1 1 3 1 1 1 1 2 1 1 1 1 1 1 2 2 1 1 2 1 1 2 1 2 1 1 2 1 2 1 1 1 28 1 1 1 3 1 1 1 4 1 3 1 313 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 2 1 1 2 2 1 1 1 1 1 96 2 1 1 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 2 2 1 1 1 1 3 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 2 1 1 2 1 1 1 1 1 1 1 1 1 2 2 185 1 1 1 1 1 1 2 1 1 2 1 2 1 1 207 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 1 3 1 1 1 1 1 2 1 1 1 1 1 1 3 2 1 1 255 1 1 1 
